cyclonedx-lib/build.xml

```diff
@@ -107,12 +107,35 @@
         <target name="jar">
                 <mkdir dir="build/jar"/>
                 <jar destfile="build/jar/temurin-gen-sbom.jar" basedir="build/classes">
-                        <manifest>
-                                <attribute name="Main-Class" value="temurin.sbom.TemurinGenSBOM"/>
-                        </manifest>
+                <manifest>
+                <attribute name="Main-Class" value="temurin.sbom.TemurinGenSBOM"/>
+                </manifest>
                 </jar>
         </target>
 
+        <target name="buildSignSBOM">
+                <mkdir dir="build/classes"/>
+                <javac srcdir="src" destdir="build/classes" classpath="build/jar/cyclonedx-core-java.jar" includeantruntime="false"/>
+                <javac debug="true" debuglevel="lines,vars,source" srcdir="src" destdir="build/classes" classpath="build/jar/cyclonedx-core-java.jar" includeantruntime="false"/>
+                </target>
+
+        <target name="runSignSBOM">
+                <exec command="openssl req -x509 -newkey rsa:2048 -keyout RSAPrivate.pem -out cert.pem -days 365 -nodes"/>
+                <exec command="openssl genrsa -des3 -out RSAPrivate.pem 2048"/>
+                <java classpath="${classpath}" classname="temurin.sbom.TemurinSignSBOM">
+                <arg value="--verbose"/>
+                <arg value="--signSBOM"/>
+                <arg value="--jsonFile"/>
+                <arg value="testSBOM.json"/>
+                <arg value="--certFile"/>
```

Comment:
Just need the private key for "signing", so just have --privateKeyFile \<private.pem\> for the --signSBOM operation
We will need a "--verifySignature" operation as well, which takes a --publicKeyFile \<public.pem\>